pkg/ignition/ignition.go

```diff
@@ -4,52 +4,59 @@ import (
 	"encoding/json"
 	"os"
 
-	ignitionConfigPrevVersion "github.com/coreos/ignition/v2/config/v3_1"
 	ignitionConfig "github.com/coreos/ignition/v2/config/v3_2"
-	"github.com/coreos/ignition/v2/config/v3_2/translate"
 	"github.com/coreos/ignition/v2/config/v3_2/types"
 	"github.com/coreos/ignition/v2/config/validate"
 	"github.com/pkg/errors"
 )
 
+type OSInterface interface {
```

Comment:
again? :) so we have an interface for each package for now? :/

Reply:
Will unify it all in a dedicated PR.

Reply:
ok. done. it is now using the `fileutil.OSInterface`.